### DISCUSSION TITLE
How to add a YT Music source?

### DISCUSSION THREAD

**Discussion by anonimo82** [TeamNewPipe/NewPipe#11816] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Hello everyone.
I didn't understand how can I use YT Music service with NewPipe, and I successfully added my favourite playlists from YouTube.
Please, can someone kindly help me?

Thanks in advance for any help, regards, have a nice day.

### Additional information

I'd like to add my Artists/Albums collection

**Comment by AudricV** [comment]
YouTube Music is not supported as a dedicated service, you can only use its search. See #4844.

However, your album/artists collection should be standard YouTube URLs. If you replace in URLs music by www, you should be able to import most of things in the app.

Note: artists should be auto-generated channels, which aren't supported the best in the app. You may have to search for the YouTube equivalents instead to get more content.
